pages/genre/genre.pixy

```diff
@@ -1,8 +1,9 @@
-component Genre(genre string, animes []*arn.Anime, user *arn.User, userScore float64)
+component Genre(genre string, animes []*arn.Anime, user *arn.User, userScore float64, userCompleted int)
 	h1(title=fmt.Sprint(len(animes)) + " anime")= strings.Title(genre)
 
-	//- if user != nil
-	//- 	h2= fmt.Sprintf("%.1f", userScore)
+	.average-score-total-completed
```

Comment:
Since this class is used only if there is a user, I think you should move it inside the if to avoid an unnecessary class in the page.